apps/worker/src/routes/notes.ts

```
@@ -1,57 +1,670 @@
 import { Hono } from 'hono';
+import { Env, NoteMessage } from '../types/env';
+import { HonoEnv } from '../types/hono';
```

ESLint

[error] 'HonoEnv' is defined but never used.

(no-unused-vars)